control-plane/plugin/src/resources/mod.rs

```diff
@@ -69,6 +70,19 @@ pub enum ScaleResources {
     },
 }
 
+/// The types of resources that support the 'SetProperty' operation.
+#[derive(clap::Subcommand, Debug)]
+pub enum SetPropertyResources {
+    /// Set volume property.
+    Volume {
+        /// ID of the volume.
+        id: VolumeId,
+        /// Max snapshot limit per volume.
+        #[clap(long)]
+        max_snapshots: u32,
```

Comment:
Shouldn't this be an enum?

Reply:
Yeah, was thinking the same, should it be rather an enum of Vol Properties then we can add more?